fuel-tx/src/builder.rs

```diff
@@ -408,7 +413,18 @@ impl<Tx: Buildable> TransactionBuilder<Tx> {
     }
 
     #[cfg(feature = "rand")]
-    pub fn add_random_fee_input(&mut self) -> &mut Self {
+    pub fn add_random_fee_input(&mut self, rng: &mut StdRng) -> &mut Self {
+        self.add_unsigned_coin_input(
+            SecretKey::random(rng),
+            rng.gen(),
+            u32::MAX as u64,
+            *self.params.base_asset_id(),
+            Default::default(),
+        )
+    }
+
+    #[cfg(feature = "rand")]
+    pub fn add_fee_input(&mut self) -> &mut Self {
```

Comment:
can this be simply 

```rust
self.add_fee_input(rand::rngs::StdRng::seed_from_u64(2322u64))
```
?

Reply:
Yes, that would work. Since it's not critical, I'll consider updating this part with next PR to `fuel-vm`.